src/flow.c

```diff
@@ -378,6 +378,89 @@ static inline void FlowUpdateEthernet(ThreadVars *tv, DecodeThreadVars *dtv,
     }
 }
 
+#if defined(ENABLE_ETM)
+#include <math.h>
+#include <float.h>
+
+inline void FlowEncryptedTrafficFinalize(const Flow *f)
+{
+    FlowSPLT *const splt = (FlowSPLT *const)&f->splt;
+    if (splt->bd_entropy || (splt->splt_count == 0))
+        return;
+
+    /* final calculation of bd variance */
+    if (splt->bd_count == 1) {
+        splt->bd_variance = 0.0;
+    } else {
+        double variance = sqrt(splt->bd_variance / (splt->bd_count - 1));
+        splt->bd_variance = (float)variance;
+    }
+
+    /* final calculation of entropy */
+    uint32_t i;
+    float entropy = 0.0;
+    for (i = 0; i < FLOW_SPLT_BD_SIZE; i++) {
```

Comment:
nit: `i` can be declared in the for loop

Reply:
Fixed with recent change